README.md

```diff
@@ -4,7 +4,12 @@
 
 [![Build Status](https://circleci.com/gh/Azure/draft.svg?style=svg)](https://circleci.com/gh/Azure/draft)
 
-_NOTE: Draft is experimental and does not have a stable release yet._
+## _NOTE: Draft is experimental and does not have a stable release yet._
+As an example, the 0.12 release moves the containment and deployment engine to your local machine. This changes quite a bit about how Draft works compared to prior releases; for details, see [the changelog](CHANGELOG.md).
```

Comment:
Oh, would it be also okay to link to https://github.com/Azure/draft/releases/tag/v0.12.0 instead of the CHANGELOG? There is a lot more context available for a user to read through the changes and why we did them.

Reply:
yes. I'll update.